thirdparty/github.go

```diff
@@ -751,3 +754,62 @@ func missingBaseRepoOwnerLogin(pr *github.PullRequest) bool {
 func missingHeadSHA(pr *github.PullRequest) bool {
 	return pr.Head == nil || pr.Head.GetSHA() == ""
 }
+
+func SendCommitQueueGithubStatus(pr *github.PullRequest, state message.GithubState, description, versionID string) error {
+	env := evergreen.GetEnvironment()
+	sender, err := env.GetSender(evergreen.SenderGithubStatus)
+	if err != nil {
+		return errors.Wrap(err, "can't get GitHub status sender")
+	}
+
+	var url string
+	if versionID != "" {
+		uiConfig := evergreen.UIConfig{}
+		if err := uiConfig.Get(env); err == nil {
+			urlBase := uiConfig.Url
```

Comment:
nit, there doesn't really seem to be a reason to define this out here rather than using directly at 770

Reply:
fixed